jme3-bullet/src/common/java/com/jme3/bullet/control/RigidBodyControl.java

```diff
@@ -266,16 +269,20 @@ public void update(float tpf) {
     public void render(RenderManager rm, ViewPort vp) {
     }
 
-    public void setPhysicsSpace(PhysicsSpace space) {
+    public void setPhysicsSpace(final PhysicsSpace space) {
+        final PhysicsSpace currentSpace = getPhysicsSpace();
```

Comment:
Either that or change your commit message to add "and a bunch of unrelated stuff because lulz."